custombackend/src/main/java/com/algolia/custombackend/views/NotNullView.java

```diff
@@ -0,0 +1,28 @@
+package com.algolia.custombackend.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+public class NotNullView extends android.support.v7.widget.AppCompatTextView {
```

Comment:
Just a comment but maybe we should package those views, if we reuse them so often in apps. 
Or maybe it's too coupled to our demos? WDYT?

Reply:
Alternatively, we might want to merge these three demos:
- Ecommerce
- QuerySuggestions
- CustomBackend

and provide a runtime setting (e.g. menubar dropdown) to enable either QS or CB

Reply:
Merging the demos is a good idea. The new Ecommerce demo Sasha designed will have all this.